app.R

```diff
@@ -0,0 +1,1382 @@
+                      plotOutput(outputId = "worldBars"),
+                      plotOutput(outputId = "worldBars2")
+                    ), 
+                    conditionalPanel(
+                      condition = "input.inputs == 'Animals'", 
+                      plotOutput(outputId = "animalPlot"), 
+                      plotOutput(outputId = "animalBars"), 
+                      plotOutput(outputId = "animalBars2")
+                    ), 
+                    conditionalPanel(
+                      condition = "input.inputs == 'Earthquakes'", 
+                      plotOutput(outputId = "quakePlot"), 
+                      plotOutput(outputId = "quakeBar"), 
+                      plotOutput(outputId = "quakeBar2")
+                    )
+                  ))
+                )
+              ),
+    tabItem(
+      tabName = "References", 
+      h2("References"), 
+      p(     #shinyBS
+        class = "hangingindent", 
+        "Bailey, E. (2015), shinyBS: Twitter bootstrap components for shiny, 
+        R package. Available from 
+        https://CRAN.R-project.org/package=shinyBS"
+      ), 
+      p(     #Boast Utilities 
+        class = "hangingindent", 
+        "Carey, R. (2019), boastUtils: BOAST Utilities, R Package.
+        Available from 
+        https://github.com/EducationShinyAppTeam/boastUtils"
+      ), 
+      p(     #shinydashboard
+        class = "hangingindent",
+        "Chang, W. and Borges Ribeio, B. (2018), shinydashboard: Create
+            dashboards with 'Shiny', R Package. Available from
+            https://CRAN.R-project.org/package=shinydashboard"
+      ), 
+      p(     #shiny
+        class = "hangingindent",
+        "Chang, W., Cheng, J., Allaire, J., Xie, Y., and McPherson, J. 
+        (2019), shiny: Web application framework for R, R Package.
+        Available from https://CRAN.R-project.org/package=shiny"
+      ), 
+      p(     #Dataset 'Countries'
+        class = "hangingindent",
+        "Countries dataset is available from
+            https://www.stat.berkeley.edu/~s133/resources.html"
+      ), 
+      p(     #shinyWidgets
+        class = "hangingindent",
+        "Perrier, V., Meyer, F., Granjon, D., Fellows, I., and Davis, W. 
+        (2020), shinyWidgets: Custom Inputs Widgets for Shiny, R package. 
+        Available from 
+        https://cran.r-project.org/web/packages/shinyWidgets/index.html"
+      ),
+      p(     #MASS
+        class = "hangingindent",
+        "Venables, W. N. and Ripley, B. D. (2002), MASS: Support Functions and 
+        Datasets for Venables and Ripley's MASS, R Package. Available from
+            https://CRAN.R-project.org/package=MASS"
+      ),
+      p(     #Attenu Earthquake
+        class = "hangingindent",
+        "Joyner, W. B., Boore, D. M., and Porcella, R. D. (1981), 
+        Peak Horizontal Acceleration and Velocity from Strong-Motion Records Including Records from the 1979 Imperial Valley California Earthquake"
+      )
```

Comment:
I believe there is a copyright insert from the style guide that is missing here that can be found on the app template.